lib/http_helper.js

```diff
@@ -1,198 +1,198 @@
     this.internal_error = error;
   }
 }
 
 class UnprocessibleEntityError extends HttpError {
-  constructor(message = "Unprocessible Entity") {
+  constructor(message = 'Unprocessible Entity') {
     super(message);
     this.code = 422;
   }
 }
 
 class UnauthorizedError extends HttpError {
-  constructor(message = "Unathorized") {
+  constructor(message = 'Unathorized') {
     super(message);
     this.code = 403;
   }
 }
 
 class NotAllowedError extends HttpError {
-  constructor(message = "This operation is not allowed.") {
+  constructor(message = 'This operation is not allowed.') {
     super(message);
     this.code = 403;
   }
 }
 
 class ConflictError extends HttpError {
-  constructor(message = "A conflict occured when trying this operation.") {
+  constructor(message = 'A conflict occured when trying this operation.') {
     super(message);
-    this.code = 409
+    this.code = 409;
   }
 }
 
 class NoFormationsFoundError extends HttpError {
-  constructor(message = "This application does not have any formations") {
-    super(message)
-    this.code = 409
+  constructor(message = 'This application does not have any formations') {
+    super(message);
+    this.code = 409;
   }
 }
 class WaitingForResourcesError extends HttpError {
   constructor(message = "We're currently waiting on resources to be created, try again in five minutes.") {
-    super(message)
-    this.code = 424
+    super(message);
+    this.code = 424;
   }
 }
 
 function clean_forward_slash(uri) {
-  if(!uri) {
+  if (!uri) {
     return uri;
   }
-  if(uri[uri.length - 1] === '/') {
+  if (uri[uri.length - 1] === '/') {
     uri = uri.substring(0, uri.length - 1);
   }
-  if(!uri.startsWith("http")) {
-    uri = "https://" + uri;
+  if (!uri.startsWith('http')) {
+    uri = `https://${uri}`;
   }
   return uri;
 }
 
 function first_match(uri, regex) {
-  var matches = (new RegExp(regex)).exec(uri);
-  if(matches && matches[1]) {
+  const matches = (new RegExp(regex)).exec(uri);
+  if (matches && matches[1]) {
     return matches[1];
   }
   return null;
 }
 
 function second_match(uri, regex) {
-  var matches = (new RegExp(regex)).exec(uri);
-  if(matches && matches[2]) {
+  const matches = (new RegExp(regex)).exec(uri);
+  if (matches && matches[2]) {
     return matches[2];
   }
   return null;
 }
 
 function third_match(uri, regex) {
-  var matches = (new RegExp(regex)).exec(uri);
-  if(matches && matches[3]) {
+  const matches = (new RegExp(regex)).exec(uri);
+  if (matches && matches[3]) {
     return matches[3];
   }
   return null;
 }
 
 function fourth_match(uri, regex) {
-  var matches = (new RegExp(regex)).exec(uri);
-  if(matches && matches[4]) {
+  const matches = (new RegExp(regex)).exec(uri);
+  if (matches && matches[4]) {
     return matches[4];
   }
   return null;
 }
 
 function fifth_match(uri, regex) {
-  var matches = (new RegExp(regex)).exec(uri);
-  if(matches && matches[5]) {
+  const matches = (new RegExp(regex)).exec(uri);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Found non-literal argument to RegExp Constructor](https://app.codacy.com/manual/Akkeris/controller-api/pullRequest?prid=5244943)